app/src/main/res/layout/activity_task_detail.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+<!--    <TextView-->
+<!--        android:id="@+id/TaskTitle"-->
+<!--        android:layout_height="wrap_content"-->
+<!--        android:layout_width="wrap_content"-->
+<!--        app:layout_constraintTop_toTopOf="parent"-->
+<!--        app:layout_constraintStart_toStartOf="parent"-->
+<!--        android:text="@string/task_detail_title"-->
+<!--        android:textColor="@color/black"-->
+<!--        android:textStyle="bold"-->
```

Comment:
Can you remove this if it's not required?

Reply:
Yup sure

Reply:
@epicadk DONE !!